### PR TITLE
Add raw data page

### PR DESCRIPTION
**Pull request by LoganHollins**
Closes bcjobs/analytics#185

### CODE REVIEW

File: app/admin/imports/route.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  model({at}) {
+    var currentDate = moment().format(DATE_FORMAT);
+    return this.store.ajax({
+      url: '/api/v1.0/reports/raw',
+      query: {
+        at: at || currentDate
+      }
+    }).then(rawData => ({
+        rawData
```
Comment (johnnyoshika) [inline app/admin/imports/route.js]: I don't think this `then` is necessary if you change `setupController` to this:

setupController(controller, rawData)

File: app/admin/template.hbs [hunk 2]
```diff
@@ -34,6 +34,11 @@
             Accounts
           {{/link-to}}
         {{/link-to}}
+        {{#link-to "admin.imports" tagName="li"}}
+          {{#link-to "admin.imports"}}
+            Daily Imports
```
Comment (johnnyoshika) [inline app/admin/template.hbs:39]: Change label to `Raw Data`

File: app/admin/imports/route.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      var body = error.responseJSON;
+      if (body && body.code) {
+        return {
+          error: body
+        };
+      }
+      throw error;
+    });
+  },
+  setupController(controller, {rawData}) {
```
Comment (johnnyoshika) [inline app/admin/imports/route.js]: You can remove `setupController` completely and work with `model.sources` in your template.

**Comment by johnnyoshika** [comment]
Instead of `imports`, let's name everything `raw data` to be consistent with Ryan's terminology. 

Ignore the above request.  Let's leave everything as `imports` if that's what Ryan wants.

File: app/admin/imports/template.hbs [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+  </div>
+</div>
+</div>
+  {{#if reports.length}}
+  <table class="table table-bordered">
+    <thead>
+      <tr>
+        <th>
+          Name
```
Comment (johnnyoshika) [inline app/admin/imports/template.hbs]: Should be `Source`

File: app/admin/imports/template.hbs [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        </th>
+        <th>
+          Amount
```
Comment (johnnyoshika) [inline app/admin/imports/template.hbs]: Should be `Total Jobs` as requested by Ryan: https://github.com/bcjobs/analytics/issues/179#issue-147481409

File: app/admin/template.hbs (hunk 2 above)
Comment (LoganHollins) [inline app/admin/template.hbs:39]: > Change label to Raw Data

`Daily Imports` was the label Ryan chose, should I still change it?

Comment (johnnyoshika) [inline app/admin/template.hbs:39]: No, if Ryan wants it to be `Daily Imports` then let's leave it.  This has me concerned though.  We use 3 different terms for the same thing:
- `Ads` in our Database and Business Layer
- `Raw` in the API
- `Daily Imports` in the UI

What do you think about asking Maxim to change `raw` to `ads` in the API layer?  At least then we can get rid of the term `raw` altogether and we will be left with only one inconsistency.

Comment (LoganHollins) [inline app/admin/template.hbs:39]: Sure, I will create an issue for Maxim.

**Comment by LoganHollins** [comment]
@johnnyoshika I have updated it with the proper naming & changes.

**Pull request merged**